package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "exosphere-sdk",
+  "version": "0.9.0",
+  "author": "Originate",
+  "bin": {
+    "exo": "cli/bin/exo"
+  },
+  "dependencies": {},
+  "description": "SDK for the Exosphere framework",
+  "devDependencies": {},
+  "engines": {
+    "npm": ">= 3.0.0"
+  },
+  "files": [
+    "cli/bin/exo",
+    "cli/bin/exo.cmd",
+    "cli/dist",
+    "exo-add/dist",
+    "exo-clone/dist",
+    "exo-create/dist",
+    "exo-deploy/dist",
+    "exo-lint/dist",
+    "exo-run/dist",
+    "exo-setup/dist",
+    "exo-sync/dist",
+    "exo-test/dist",
+    "exosphere-shared/dist"
```

Comment:
Won't we also need the `exosphere-shared/templates` folder available so that `exo-add`, `exo-create`, and `exo-setup` can use it?